### PR TITLE
Bump source looker connector version

### PR DESCRIPTION
**Pull request by tuliren**
## What
- Follow up PR for #3911.

## Pre-merge Checklist
Expand the checklist which is relevant for this PR. 

<summary> <strong> Connector checklist </strong> </summary>
<p>

- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [x] Secrets are annotated with `airbyte_secret` in output spec
- [x] Unit & integration tests added as appropriate (and are passing)
    * Community members: please provide proof of this succeeding locally e.g: screenshot or copy-paste acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] `/test connector=connectors/<name>` command as documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
    * Community members can skip this, Airbyters will run this for you. 
- [x] Code reviews completed
- [x] Credentials added to Github CI if needed and not already present. [instructions for injecting secrets into CI](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#using-credentials-in-ci). 
- [x] Documentation updated 
    - [x] `README.md`
    - [x] `docs/SUMMARY.md` if it's a new connector
    - [x] Reference docs in the `docs/integrations/` directory.
    - [x] Changelog in the appropriate page in `docs/integrations/...`. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] Build status added to [build page](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/builds.md)
- [x] Build is successful
- [x] Connector version bumped like described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [x] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [x] No major blockers
- [x] PR merged into master branch
- [x] Follow up tickets have been created
- [x] Associated tickets have been closed & stakeholders notified
</p>

### CODE REVIEW

File: docs/integrations/sources/looker.md [hunk 1]
```diff
@@ -74,3 +74,14 @@ The Looker connector should not run into Looker API limitations under normal usa
 
 Please read the "API3 Key" section in [Looker's information for users docs](https://docs.looker.com/admin-options/settings/users) for instructions on how to generate Client Id and Client Secret.
 
+## CHANGELOG
+
+| Version | Date | Pull Request | Subject |
```
Comment (sherifnada) [inline docs/integrations/sources/looker.md:79]: 👏🏼👏🏼 👏🏼

**Review by sherifnada — APPROVED** [review]
Beautiful! Can you create a follow up issue to add testing for the new feature? I believe we'll need to create a "Look" in our looker sandbox and reference its ID in our tests for this feature to be tested consistently

**Comment by tuliren** [comment]
Added the issue: #4353.

**Pull request merged**